src/pages/base-page.ts

```diff
@@ -1,4 +1,5 @@
-import { Browser, DefaultTimeout, ElementArrayFinder, ElementFinder } from "../wdio";
+import {Browser, DefaultTimeout, ElementArrayFinder, ElementFinder, LocatorType} from '../wdio';
```

Comment:
Changed back to the previous format manually. It was strange because I merely opened WebStorm the same way I do for the project's FE project

Reply:
Probably because project configuration is different from the one used on this library. Notice also that imports are now diffing in the type of quotes used. They should be all in double quotes.

Reply:
In the end it was not Prettier. Set back to single quotes in imports, etc. thanks to @fjaguado